qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import logging
+import requests
+import time
+
+from .helper import DashboardTestCase, JObj, JList, JLeaf
+
+logger = logging.getLogger(__name__)
+
+
+class MgrModuleTestCase(DashboardTestCase):
+    @classmethod
+    def tearDownClass(cls):
+        cls._ceph_cmd(['mgr', 'module', 'disable', 'telemetry'])
+        super(MgrModuleTestCase, cls).tearDownClass()
+
+    def wait_until_rest_api_accessible(self):
+        """
+        Wait until the REST API is accessible.
+        """
+
+        def _check_connection():
+            try:
+                # Try to reaching an API endpoint successfully.
+                self._get('/api/mgr/module')
+                if self._resp.status_code == 200:
+                    return True
+            except requests.ConnectionError:
+                pass
```

Comment:
You could return `False` here instead of doing nothing (`pass`) and then returning on the next line.

Reply:
In this case the function will return ``False`` only if an exception is thrown. But this is not the intention of this function.